01-Authorization-RS256/go.mod

```diff
@@ -0,0 +1,12 @@
+module 01-Authorization-RS256
+
+go 1.17
+
+require (
+	github.com/auth0/go-jwt-middleware v1.0.1
+	github.com/codegangsta/negroni v1.0.0
```

Comment:
You might want to update the reference to https://github.com/urfave/negroni
Or, use Gin, which is way more popular nowadays.

Reply:
That's a good suggestion and will definitely tackle that in the upcoming PRs so we perform these library changes in isolation. Thanks Rita:) 

The same for [github.com/form3tech-oss/jwt-go](github.com/form3tech-oss/jwt-go) (replacing it with https://github.com/golang-jwt/jwt)